packages/html-spec/src/forms/forms.json

```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "component": "forms",
+        "selector": [
+            ".k-form"
+        ],
+        "states": [
+            {
+                "name": "error",
+                "selector": [
+                    ".k-form-error"
+                ]
+            }
+        ],
```

Comment:
This is not correct. The class `.k-form-field-error` is added to the form field element if an item of the form is in error state. Thus, this should relate to the form field options.